conjure_oxide/tests/generated_tests.rs

```diff
@@ -2,11 +2,10 @@ use conjure_oxide::parse::parse_json;
 
 fn integration_test(path: &str) {
     let mut cmd = std::process::Command::new("conjure");
-    cmd.arg("pretty --output-format=astjson")
-        .arg(format!(
-            "{path}/input.essence > {path}/input.astjson.json",
-            path = path
-        ));
+    cmd.arg("pretty --output-format=astjson").arg(format!(
+        "{path}/input.essence > {path}/input.astjson.json",
+        path = path
+    ));
     cmd.output().unwrap();
```

Comment:
Rust tests can return Result<>, so you can use ? instead of unwrap.

See https://doc.rust-lang.org/book/ch11-01-writing-tests.html#using-resultt-e-in-tests